Cargo.toml

```diff
@@ -40,7 +41,7 @@ tracing-indicatif = "0.3.9"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 url = "2.5.4"
 walkdir = "2.5.0"
-wdl = { version = "0.14.0", features = ["cli", "codespan", "engine", "lsp"], git = "https://github.com/stjude-rust-labs/wdl", rev = "ccc8850" }
+wdl = { version = "0.14.0", features = ["cli", "codespan", "engine", "lsp", "doc"], git = "https://github.com/stjude-rust-labs/wdl", branch = "a-frantz/wdl-doc" }
```

Comment:
I know you're already aware, but I'd like to hang a comment thread here to make sure we get back to a mainline commit before merging